config/acme/config_files.xml

```diff
@@ -144,30 +144,6 @@
     <desc>file containing specification of archive files for each component (for documentation only - DO NOT EDIT)</desc>
     <schema>$CIMEROOT/config/xml_schemas/config_archive.xsd</schema>
   </entry>
-  <entry id="ARCHIVE_SPEC_FILE">
```

Comment:
You need ARCHIVE_SPEC_FILE with the updated paths.  Compare to the cesm record of the same variable.